### DISCUSSION TITLE
Unable to remove hb-blog-sidebar-taxonomies hook

### DISCUSSION THREAD

**Discussion by LITUATUI** [hbstack/theme-cards#270] [Q&A] [answered]
I don't want taxonomies in the sidebar, but if I remove the hb-blog-sidebar-taxonomies hook from params.yaml the website becomes disfigured (no background image, no header nor footer, no fixed sidebars, etc).

What I'm doing now is removing it from module.yaml but then I get this warning in the terminal when running hugo serve:

`WARN  [hugopress] [hb-blog-sidebar-taxonomies] hook does't exist: layouts/partials/hugopress/modules/hb-blog-sidebar-taxonomies/hooks/hb-blog-sidebar.html`

Is this expected behavior?

**Comment by razonyang** [comment]
You'll need to remove it from params.yaml as well.

https://github.com/LITUATUI/pushevs.com/blob/main/config%2F_default%2Fparams.yaml#L192-L196

I think it was there to show how to change the order of sidebar modules.

**Reply by LITUATUI** [reply]
Yes, I wrote hugo.yaml but wanted to write params.yaml.

The disfigured website happens after removing the hook from params.yaml.

**Reply by razonyang** [reply, marked as answer]
I dont see any problem on your site (latest commit) after deleting those lines.

![image](https://github.com/hbstack/theme-cards/assets/17720932/93924f48-38f8-441f-8c15-3dcab204e9b3)

**Reply by LITUATUI** [reply]
Sorry @razonyang it works now, don't know what happen.

**Reply by razonyang** [reply]
No problem, perhaps just a cache issue that requires restarting server.
